packaging/nnstreamer.spec

```diff
@@ -875,6 +875,13 @@ cp -r result %{buildroot}%{_datadir}/nnstreamer/unittest/
 %{_prefix}/lib/nnstreamer/filters/libnnstreamer_filter_openvino.so
 %endif
 
+%if 0%{?mxnet_support}
+%files mxnet
+%manifest nnstreamer.manifest
+%license LICENSE
+%{_prefix}/lib/nnstreamer/filters/libnnstreamer_filter_mxnet.so
+%endif
```

Comment:
This is incomplete. You don't have subpackage definition.

You may skip adding mxnet in spec file unless you are going to contribute MXNET packaging to Tizen

Reply:
I see, I will remove it too.